lib/command.js

```diff
@@ -62,14 +60,11 @@ module.exports = function (yargs, usage, validation) {
         if (~context.files.indexOf(joined)) return visited
         // keep track of visited files in context.files
         context.files.push(joined)
-        // map "command path" to the directory path it came from
-        // so that dir can be relative in the API
-        context.dirs[context.commands.concat(parseCommand(visited.command || commandFromFilename(filename)).cmd).join('|')] = dir
```

Comment:
I love how much this simplifies things; except for the fact that yargs' documentation is terrifyingly verbose at this point, I'm really feeling like the command functionality is knocking it out of the park -- phew, that was hard work wasn't it.